nautobot/extras/tests/test_jobs.py

```diff
@@ -762,7 +769,7 @@ def test_job_fires_webhooks(self, mock_enqueue_webhooks):
         webhook.content_types.set([status_ct])
 
         job_result = create_job_result_and_run_job(module, name)
-        self.assertEqual(job_result.status, JobResultStatusChoices.STATUS_SUCCESS)
+        self.assertJobResultStatus(job_result, JobResultStatusChoices.STATUS_SUCCESS)
```

Comment:
A lot of cases like this in test_*.py
```suggestion
        self.assertJobResultStatus(job_result)
```